lib/private/handlers/container-images.ts

```diff
@@ -94,7 +99,10 @@ export class ContainerImageAssetHandler implements IAssetHandler {
     }
 
     this.host.emitMessage(EventType.UPLOAD, `Push ${initOnce.imageUri}`);
-    await dockerForPushing.push({ tag: initOnce.imageUri, quiet: this.options.quiet });
+    await dockerForPushing.push({
+      tag: initOnce.imageUri,
+      subprocessOutputDestination: this.options.subprocessOutputDestination,
```

Comment:
Since we don't need to pass `shellEventPublisher` here (I guess it already got passed upon initialiazing `dockerForPushing`)... can't we also pass `subprocessOutputDestination` yonder?

Reply:
Please respond to my question?

Reply:
Yes sorry I just didn't get a chance to push this, absolutely right that it also can be passed "yonder" 😂